client/src/pages/SignUp/useStyles.ts

```diff
@@ -1,25 +1,41 @@
 import { makeStyles } from '@mui/styles';
+import { theme } from '../../themes/theme';
 
 const useStyles = makeStyles(() => ({
   root: {
-    minHeight: '100vh',
     '& .MuiInput-underline:before': {
       borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
     },
   },
   authWrapper: {
-    display: 'flex',
-    alignItems: 'flex-start',
-    justifyContent: 'space-between',
-    flexDirection: 'column',
     minHeight: '100vh',
-    paddingTop: 23,
+    paddingTop: 20,
+    backgroundColor: '#F5F5F5',
+  },
+  formContainer: {
+    boxShadow: 'rgba(0, 0, 0, 0.24) 0px 3px 8px',
+    height: '100%',
+    maxHeight: '600',
+    borderRadius: 8,
+    margin: 'auto',
+    backgroundColor: 'white',
   },
   welcome: {
-    fontSize: 26,
-    paddingBottom: 20,
+    fontSize: 24,
+    fontWeight: theme.typography.button.fontWeight,
```

Comment:
We could set font weights to typography variants rather than using the button font weight in most cases